website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -12,23 +12,48 @@ displayed_sidebar: "docs"
+dbt Labs is committed to providing backward compatibility for all versions 1.x, with the exception of any changes explicitly mentioned below. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
 
-The **Keep on latest version** setting is currently available in beta for select dbt Cloud customers, rolling out to wider availability through February and March.
+The dbt Core v1.8.0-b1 release of dbt-core & dbt Labs-maintained adapters is planned for February 28th.
 
-For users of dbt Core, the v1.8.0-b1 release of `dbt-core` & dbt Labs-maintained adapters is planned for February 28.
+## Keep on the latest version in dbt Cloud
 
-:::
+ With dbt Cloud, you can get early access to many new features and functionality before it is in the GA release of dbt Core v1.8 without the need to manage version upgrades. For more details, refer to [Upgrade Core version in Cloud](/docs/dbt-versions/upgrade-dbt-version-in-cloud).
 
-## What to know before upgrading
-
-dbt Labs is committed to providing backward compatibility for all versions 1.x, with the exception of any changes explicitly mentioned below. If you encounter an error upon upgrading, please let us know by [opening an issue](https://github.com/dbt-labs/dbt-core/issues/new).
+The **Keep on latest version** setting is currently available in closed beta for select dbt Cloud customers and will have wider availability in March of 2024.
 
 ## New and changed features and functionality
 
-Features and functionality new in dbt v1.8
+Features and functionality new in dbt v1.8.
+
+### New dbt Core adapter installation procedure
+
+Before v1.8, when you installed an adapter, you would automatically get `dbt-core ` installed along with the adapter package (if you didn’t already have an existing, compatible version of dbt-core).
+
+Beginning in v1.8, you must install _both_ dbt-core and the desired adapter. A new `pip` installation needs to look like this:
+
+```shell
+pip install dbt-core dbt-ADAPTER_NAME
+```
+
+### Unit Tests
+
+Historically, dbt's test coverage was confined to [“data” tests](/docs/build/data-tests), assessing the quality of input data or resulting datasets' structure.
+
+In `1.8`, we're introducing native support for [unit testing](/docs/build/unit-tests). Unit tests validate your SQL modeling logic on a small set of static inputs __before__ you materialize your full model in production. They support a test-driven development approach, improving both the efficiency of developers and reliability of code.
+
+When you execute `dbt test` in v1.8+, it will run both unit and data tests. Use the [`test_type`](/reference/node-selection/methods#the-test_type-method) method to run only unit or data tests:
```

Comment:
```suggestion
Starting from v1.8, when you execute the `dbt test` command, it will run both unit and data tests. Use the [`test_type`](/reference/node-selection/methods#the-test_type-method) method to run only unit or data tests:
```